_data/simple-icons.json

```
@@ -6259,6 +6259,11 @@
             "title": "Tata",
             "hex": "486AAE",
             "source": "https://www.tata.com/"
+        },
+		{
+            "title": "TeamCity",
+            "hex": "000000",
+            "source": "https://www.jetbrains.com/company/brand/logos/"
```

A small formatting issue that should be fixed before this can be merged

```suggestion
        },
        {
            "title": "TeamCity",
            "hex": "000000",
            "source": "https://www.jetbrains.com/company/brand/logos/"
```

---

Done!